### PR TITLE
Implement ck_pr_dec_is_zero family of functions

### PR DESCRIPTION
**Pull request by postwait**

### CODE REVIEW

**Comment by pkhuong** [comment]
I'd rather see ck_pr_##K##_##S##_is_zero be implemented in terms of ck_pr_##K##_##S##_zero (or vice versa). This will make it easier to implement both in a wait-free manner on architectures that support it (e.g., x86).

**Comment by pkhuong** [comment]
Did some more work on this at https://github.com/pkhuong/ck/tree/ck_pr_dec_is_zero . Feel free to pull in what makes sense to you!

**Comment by postwait** [comment]
PR updated with your changes

**Comment by pkhuong** [comment]
@sbahra this is accidentally kind of invasive. Feel like taking it for a spin on the gcc farm? Otherwise, my concerns are all addressed.

File: doc/ck_pr_inc [hunk 1]
```diff
@@ -29,22 +29,31 @@
...
+.Nm ck_pr_inc_int_is_zero ,
 .Nm ck_pr_inc_64 ,
 .Nm ck_pr_inc_64_zero ,
+.Nm ck_pr_inc_64_is_zero ,
 .Nm ck_pr_inc_32 ,
 .Nm ck_pr_inc_32_zero ,
+.Nm ck_pr_inc_32_is_zero ,
 .Nm ck_pr_inc_16 ,
-.Nm ck_pr_inc_16_zero ,
+.Nm ck_pr_inc_32_zero ,
```
Comment (pkhuong) [inline doc/ck_pr_inc]: womp. bad copy pasta here.

**Pull request merged**